src/transformer.js

```diff
@@ -295,11 +309,14 @@
         }
         if (token.value === 'if') {
           var func = fstats.functions[token.ownerFuncToken.white];
-          token.isIfKeyword = true;
-          token.ownerCountIndex = func.ifs.length;
-          token.ownerCountType = transformer.typeIf;
-          token.ownerCountFuncId = token.ownerFuncToken.white;
-          func.ifs.push(0);
+          //TODO For what case will 'func' become undefined? Pls investigate.
```

Comment:
Thanks. This issue is now fixed. Please rebase on top of master to make sure some of your other concerns have been addressed now.